.github/workflows/install-with-rancher.yaml

```diff
@@ -1,6 +1,11 @@
-name: Install ROS on a server through Rancher
+name: Bootstrap RancherOS on a VM with Rancher
 
 on:
+  on:
+  pull_request:
```

Comment:
why not on branches: master so its only triggered when something is merged onto master

Reply:
Yes could be, that was mainly to avoid executing it if a direct push is done to `master`. But no problem to change :)

We can also trigger this each night in addition or instead of push on `master` branch, don't know exactly what you prefer guys ;-)

Reply:
I dont think there is a need to have it on PR closed, just when something is merged to master. There should not be anyone merging directly to master without opening a PR first :)

Reply:
And that means you can remove the if below and having as a straight job

Reply:
OK, so I changed to schedule the test when a push is done on `master`, it should be fine now.